src/udiskslinuxdriveata.c

```diff
@@ -596,9 +618,9 @@ udisks_linux_drive_ata_refresh_smart_sync (UDisksLinuxDriveAta  *drive,
     {
       guchar count;
       gboolean noio = FALSE;
-      if (!get_pm_state(device, error, &count))
+      awake = udisks_linux_drive_ata_is_active(device, error, &state);
+      if (&error != NULL)
```

Comment:
So this would result in `GError ***` data type.

Also, `error` may be NULL as per common glib calling conventions and in that case you couldn't determine whether the call succeeded or not - that's why the `gboolean` return value is used.